worlds/yugioh06/__init__.py

```diff
@@ -0,0 +1,538 @@
+import logging
+import os
+import sys
```

Comment:
```suggestion
import os
```
Remove unused imports